packaging/tools/style-format.sh

```diff
@@ -70,6 +72,14 @@ for f in $*; do
         stylename="C"
     fi
 
+    if [[ $lang == c && $clang_version_warning == 0 ]]; then
```

Comment:
No reason to check this for each file, add the check above the loop, preferably on line 23, and I'd change it to erroring out instead of just warning.
Error messages should be formatted as:
"$0: clang-format version 10 or 11 required"

Reply:
I've moved it to that line and erroring out.
The original logic for having it inside the loop was that, we should allow make style-check-(changed) to run on things like python files without actually needing clang-format. But I think that might be a rare case to have someone need to check just non c/cpp files.